alacritty.yml

```diff
@@ -534,6 +573,57 @@
 # If the same trigger is assigned to multiple actions, all of them are executed
 # at once.
 #key_bindings:
+  #- { key: Paste,                    action: Paste                            }
+  #- { key: Copy,                     action: Copy                             }
+  #- { key: L,         mods: Control, action: ClearLogNotice                   }
+  #- { key: L,         mods: Control, chars: "\x0c"                            }
+  #- { key: PageUp,    mods: Shift,   action: ScrollPageUp,   mode: ~Alt       }
+  #- { key: PageDown,  mods: Shift,   action: ScrollPageDown, mode: ~Alt       }
+  #- { key: Home,      mods: Shift,   action: ScrollToTop,    mode: ~Alt       }
+  #- { key: End,       mods: Shift,   action: ScrollToBottom, mode: ~Alt       }
+
+  # Keyboard Motion
+  #- { key: Escape, mods: Control, mode: KeyboardMotion, action: ScrollToBottom     }
+  #- { key: Escape, mods: Control,                       action: ToggleKeyboardMode }
+  #- { key: Escape,                mode: KeyboardMotion, action: ScrollToBottom     }
+  #- { key: Escape,                mode: KeyboardMotion, action: ToggleKeyboardMode }
```

Comment:
It seems a bit strange that we have both `ESC + Control` + `ESC` to leave `KeyboardMotion` mode.

Reply:
`Esc` is the standard vi bind and `Ctrl + Esc` is the bind we use to enter this mode.

I think it makes a lot of sense to bind both. I don't know which one I'd remove if I would decide to remove one tbh.